workbench/controllers/RestResponseInstrumenter.php

```diff
@@ -90,4 +95,21 @@ function instrument($s) {
         return preg_replace('@' . $this->pattern . '@', $this->replacement, $s);
     }
 }
-?>
\ No newline at end of file
+
+class RegexUrlInsturmentation implements Insturmentation {
```

Comment:
why is this special for urls? seems like it's special that it takes a callback. perhaps rename the class or the param?